### PR TITLE
Update batchnorm doc and change default.

### PR DESCRIPTION
**Pull request by cottrell**
I'm hoping this triggers a build/test or something.

Will have to read more about if not.

https://github.com/tensorflow/tensorflow/issues/33027#issuecomment-542402972

### CODE REVIEW

**Comment by rthadur** [comment]
Seems you have checked out lot of files , it will be very hard to resolve the conflicts , please submit a new PR.

**Comment by cottrell** [comment]
@googlebot I signed it!

**Comment by cottrell** [comment]
@rthadur weird ... there should only be one small change in one file. I'll have a look.
